packages/client/src/views/RecordAudit/RecordAudit.test.tsx

```diff
@@ -161,10 +167,16 @@ describe('Record audit summary for a draft death declaration', () => {
   })
 
   it('Check values for saved declarations', async () => {
-    expect(component.find('#status_value').hostNodes().text()).toBe('Draft')
-    expect(component.find('#type_value').hostNodes().text()).toBe('Death')
-    expect(component.exists('#drn_value')).toBeFalsy()
-    expect(component.find('#placeOfDeath_value').hostNodes()).toHaveLength(1)
+    expect(
+      component.find({ 'data-testid': 'status-value' }).hostNodes().text()
+    ).toBe('Draft')
```

Comment:
@Zangetsu101 do you think it's a good idea to use `data-testid`'s instead of # id's? It goes little bit against the grain on our current convention but generally think it could make more sense

Reply:
Agreed. We're only using the id's for tests so yeah `data-testid` makes the intent clearer